hack/verify-boilerplate.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
```

Comment:
It would be pretty cool if these boilerplate.py & verify-boilerplate.sh had a comment indicating which repo they came from. Do you mind making an upstream PR there if you feel this is right?

Reply:
I'm not sure they originally came from there. But I can investigate if I find the real authors.

Reply:
So the scripts actually originate from the k8s.io/kubernetes repo. I added a notice as suggested.